apps/src/sites/code.org/pages/views/theme_google_analytics.js

```diff
@@ -1 +1,15 @@
 import '@cdo/apps/metrics/GoogleAnalyticsReporter';
+
+// Google Tag Manager
+(function (w, d, s, l, i) {
+  w[l] = w[l] || [];
+  w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
+  var f = d.getElementsByTagName(s)[0],
+    j = d.createElement(s),
+    // eslint-disable-next-line eqeqeq
+    dl = l != 'dataLayer' ? '&l=' + l : '';
+  j.async = true;
+  j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
+  f.parentNode.insertBefore(j, f);
+})(window, document, 'script', 'dataLayer', 'GTM-TZZBRK5');
+// End Google Tag Manager
```

Comment:
Since we only want this on code.org I added it here instead of `GoogleAnalyticsReporter.js`.

Reply:
This seems reasonable for now.  If we do end up using GTM in the same places, it would be good to move it there.